portara-tools/portara-website
# Prep for deployment

**Problem**
- Deploy would not go through to Heroku. Could not find the culprit, but we need to deploy to check functionality
**Solution**
- Had to do with NODE_ENV variables. Had to only use dotenv when NODE_ENV is set to dev. Also had to change various hardcoded urls to work with deployed version instead.
**To test**
- GH OAuth has been tested and works. Go to heroku and our temp domain, and try logging in with oauth. It should re-direct back to homepage.
